Intersect (Core)/Extensions/EnumerableExtensions.cs

```diff
@@ -39,5 +44,77 @@ public static IEnumerable<T> SelectManyRecursive<T>(this IEnumerable<T> source,
 
             return result.Any() ? result.Concat(result.SelectManyRecursive(selector, cancellationToken)) : result;
         }
+
+        public static string GetDescription(this Enum value)
+        {
+            Type type = value.GetType();
+            string name = Enum.GetName(type, value);
+            if (name != null)
+            {
+                FieldInfo field = type.GetField(name);
+                if (field != null)
+                {
+                    DescriptionAttribute attr =
+                           Attribute.GetCustomAttribute(field,
+                             typeof(DescriptionAttribute)) as DescriptionAttribute;
+                    if (attr != null)
+                    {
+                        return attr.Description;
+                    }
+                }
+            }
+            return string.Empty;
+        }
+
+        public static string[] GetDescriptions(Type enumType, string except = null)
+        {
+            return Enum.GetValues(enumType)
+                .Cast<Enum>()
+                .Select(value => value.GetDescription())
+                .Where(value => string.IsNullOrEmpty(except) || value != except)
+                .ToArray();
+        }
```

Comment:
These aren't used anymore right? i.e. Can  be deleted?

Reply:
yes, done